cli/copr_cli/main.py

```diff
@@ -1243,6 +1256,11 @@ def setup_parser():
     cli_use_output_format(parser_builds, default=None)
     parser_builds.set_defaults(func="action_list_builds")
 
+    parser_get = subparsers.add_parser("get", help="Get information about project")
+    parser_get.add_argument("owner", help="Owner of a package. Groups begins with `@` e.g. @copr")
+    parser_get.add_argument("project", help="Name of the project.")
```

Comment:
I think this will require us to call the command as `copr-cli get @copr copr` instead of `copr-cli get @copr/copr`? That would be inconsistent with other commands.

Reply:
Hmm I didn't think of calling it as `@copr/copr`... It's more intuitive this way